src/main/resources/zenodo-1.0.0-swagger-2.0.yaml

```diff
@@ -566,6 +663,71 @@ definitions:
           type: string
         example:
           conceptdoi: https://doi.org/10.5281/zenodo.705645
+      modified:
+        type: string
+        format: date-time
+      owner:
+        type: integer
+      record_id:
+        type: integer
+      state:
+        type: string
+      status:
+        type: string
+        description: oddly, this field is populated in a list of records but not when retriving individual deposits
+      submitted:
+        type: boolean
+      title:
+        type: string
+  Deposit:
+    type: object
+    properties:
+      conceptrecid:
+        type: integer
+      created:
+        type: string
+        format: date-time
+      files:
+        type: array
+        items:
+          $ref: '#/definitions/DepositionFile'
+      id:
+        type: integer
+      links:
+        type: object
+        properties:
+          conceptdoi:
+            type: string
+          self:
+            type: string
+          html:
+            type: string
+          badge:
+            type: string
+          files:
+            type: string
+          bucket:
+            type: string
+          thumb250:
+            type: string
+          thumbs:
```

Comment:
`thumbs` is the breaking change, a new field that is a map instead of a single string